data/INRA/variables.ttl

```diff
@@ -0,0 +1,35 @@
+
+bdg:concentration a qb:MeasureProperty, sosa:ObservableProperty;
+  rdfs:label "Concentration";
+  rdfs:notation "Concentration";
+  sosa:hasFeatureOfInterest owl:Thing;
```

Comment:
The feature of interest is already attached on the dataset level, no need to repeat here